cmd/reinstall.go

```diff
@@ -28,23 +28,26 @@ func (h *Hammer) fetchMachine(machineID string) (*models.ModelsV1MachineResponse
 
 // wipe only the disk that has the OS installed on one of its partitions, keep all other disks untouched
 func (h *Hammer) reinstall(m *models.ModelsV1MachineResponse, hw *models.DomainMetalHammerRegisterMachineRequest, eventEmitter *event.EventEmitter) (*kernel.Bootinfo, error) {
+	if m.Allocation.BootInfo == nil {
```

Comment:
can we check this already at metal-api to prevent reinstallation at all from the beginning ?

Reply:
This is good :+1: